imgui_draw.cpp

```diff
@@ -1789,6 +1793,107 @@ bool ImFontAtlas::GetMouseCursorTexData(ImGuiMouseCursor cursor_type, ImVec2* ou
     return true;
 }
 
+static int IMGUI_CDECL FloatReverseComparer(const void* lhs, const void* rhs)
```

Comment:
Should this comparer clamp differences < 1.0f to zero? This render 0.24f and -0.7f equal.
<https://godbolt.org/z/xfm82H>

If that is an error, maybe this will work better:
```cpp
static int IMGUI_CDECL FloatReverseComparer(const void* lhs, const void* rhs)
{
    const float diff = *(float*)rhs - *(float*)lhs;
    if (diff > 0.0f)
        return 1;
    else if (diff < -0.0f)
        return -1;
    else
        return 0;
}
```

Reply:
You are right. Thank you for looking into the code :+1: